sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -475,17 +473,16 @@ def test_read_from_query_unflatten_records(self):
     self.assertFalse(reader.flatten_results)
 
   def test_using_both_query_and_table_fails(self):
-    with self.assertRaises(ValueError) as exn:
+    with self.assertRaisesRegexp(
+        ValueError,
+        r'Both a BigQuery table and a query were specified. Please specify only'
+        r' one of these.'):
```

Comment:
<!--new_thread; commit:4709161da347f4fa5eedbd7f3c09911b573fadac; resolved:0-->
Let's add `^` and `$` and escape the '.' characters throughout.

Reply:
I don't agree with adding ^ and $ everywhere. Checking for long stretches of text is brittle and I don't want to make this check more brittle.
I'll go ahead with removing/escaping special regex characters like `.`.

Reply:
Thanks, I agree this is brittle and suggested this since the original code tested for equality.  Seems okay either way.